README.md

```diff
@@ -0,0 +1,24 @@
+# PGXMan Buildkit
+
+PGXMan Buildkit is a YAML configuration file that `pgxman` employs to define the build and packaging of a PostgreSQL extension.
+Explore the specification for more details [here](https://github.com/pgxman/pgxman/blob/main/spec/buildkit.md).
```

Comment:
some suggestions:
* move the spec to this repo - cant read it in `pgxman`
* provide a simple / short example here in README ?
* provide a complete "documented" (e.g. with YAML comments) example in the spec or in a separate file (most people dont want to actually read the spec 😁 )

Reply:
Some examples for reference: 
https://render.com/docs/blueprint-spec
https://devcenter.heroku.com/articles/build-docker-images-heroku-yml
https://devcenter.heroku.com/articles/app-json-schema

Reply:
This is good feedback. I added the following:

* buildkit spec is moved to this repo
* add a short example to RAEDME
* add a complete example to the spec